template/drush/README.md

```diff
@@ -1,3 +1,13 @@
+# Drush configuration and aliases
+
 This directory is intended to contain drush configuration that is not site or environment specific.
 
 Site specific drush configuration lives in `sites/[site-name]`.
+
+## Site aliases
+
+You should add the Drush aliases for any remote environments (such as Acquia Cloud) to the `site-aliases` directory. This allows developers to access remote environments using simple aliases such as `drush @mysite.dev uli`. Note that if you are using Acquia Cloud, developers can also download these aliases manually from their Insight account, but providing them with the project makes everyone’s life a little easier.
```

Comment:
I also find it helpful to define a drush alias for the local environment (e.g. `site.local`), as that allows all commands to always be accessed through a familiar syntax/alias structure; also it allows people to have a local alias with differing paths/config more easily.